controls/ubuntu-18.04-server-cis-5.2.15.rb

```diff
@@ -39,14 +39,16 @@
   "
   impact 0.5
   tag severity: "medium"
-  tag gtitle: nil
-  tag gid: nil
-  tag rid: nil
-  tag stig_id: nil
-  tag fix_id: nil
-  tag cci: nil
   tag nist: ["SC-8", "Rev_4"]
   tag cis_level: 1
   tag cis_controls: ["14.4", "Rev_7"]
   tag cis_rid: "5.2.15"
+  cfg = parse_config_file('/etc/ssh/sshd_config', { assignment_regex: /^\s*(\S*)\s*(.*?)\s*$/ } )
+  describe cfg do
+    its('KexAlgorithms') { should be }
```

Comment:
should be what?

Reply:
Again, this is intended to ensure that this value is set explicitly.